.github/workflows/workflow-executor.yml

```diff
@@ -0,0 +1,74 @@
+name: Reusable Workflow executing Makefile to execute the CI pipeline for workflows
+
+on:
+  workflow_call:
+    inputs:
+      workflow_id:
+        required: true
+        type: string
+      push_pr:
+        required: false
+        type: boolean
+        default: true
+
+env:
+  # TODO Update to parodos-dev
+  DEPLOYMENT_REPO: dmartinol/serverless-workflows-helm
+  REGISTRY_REPO: orchestrator
+  GH_TOKEN: ${{ secrets.HELM_REPO_TOKEN }}
+jobs:
+  build-and-push-image:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - name: Build
+      run: |
+        make WORKFLOW_ID=${{ inputs.workflow_id }} \
+          REGISTRY_REPO=${{ env.REGISTRY_REPO }} \
+          REGISTRY_USERNAME=${{ secrets.NEW_QUAY_USERNAME }} \
+          REGISTRY_PASSWORD=${{ secrets.NEW_QUAY_PASSWORD }} \
+          push-image
```

Comment:
added comment below, but it would be more descriptive to have the target specified here, e.g.:
`build-image push-image`